docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_w_image.scss

```diff
@@ -101,7 +101,11 @@ $hb-sidebar-width: 16.5rem;
       @supports (object-fit: cover) {
         object-fit: cover;
         width: 100%;
-        max-height: 250px;
+        max-height: 23.4375rem;
```

Comment:
hmm yeah I still wonder if we shouldn't just let this be 100% height to allow regular image proportions and not crop people's heads off.

Reply:
The concern with having a max-height of 100% is that they might have a tall enough image that pushes all text off the screen when scrolling. What if we try a max-height of `90vh` to ensure that there is always some text or part of the page visible?